cosmo_tester/test_suites/image_based_tests/hello_world_backwards_test.py

```diff
@@ -1,43 +0,0 @@
-########
```

Comment:
These have been moved into the hello world tests and rationalised so that they're testing backwards compatibility of cloudify types versions rather than just extra arbitrary blueprints.